packages/mdc-ripple/index.ts

```diff
@@ -26,88 +26,80 @@ import MDCRippleAdapter from './adapter';
-   */
-  static createAdapter(instance) {
+  static createAdapter(instance: RippleCapableSurface): MDCRippleAdapter {
     const MATCHES = util.getMatchesProperty(HTMLElement.prototype);
 
     return {
-      browserSupportsCssVars: () => util.supportsCssVariables(window),
-      isUnbounded: () => instance.unbounded,
-      isSurfaceActive: () => instance.root_[MATCHES](':active'),
-      isSurfaceDisabled: () => instance.disabled,
       addClass: (className) => instance.root_.classList.add(className),
-      removeClass: (className) => instance.root_.classList.remove(className),
-      containsEventTarget: (target) => instance.root_.contains(target),
-      registerInteractionHandler: (evtType, handler) =>
-        instance.root_.addEventListener(evtType, handler, util.applyPassive()),
+      browserSupportsCssVars: () => util.supportsCssVariables(window),
+      computeBoundingRect: () => instance.root_.getBoundingClientRect(),
+      containsEventTarget: (target) => instance.root_.contains(target as Node),
+      deregisterDocumentInteractionHandler: (evtType, handler) =>
+        document.documentElement.removeEventListener(evtType, handler, util.applyPassive()),
       deregisterInteractionHandler: (evtType, handler) =>
         instance.root_.removeEventListener(evtType, handler, util.applyPassive()),
+      deregisterResizeHandler: (handler) => window.removeEventListener('resize', handler),
+      getWindowPageOffset: () => ({x: window.pageXOffset, y: window.pageYOffset}),
+      isSurfaceActive: () => {
+        const root = instance.root_;
+        return root[MATCHES as 'matches'](':active');
+      },
+      isSurfaceDisabled: () => Boolean(instance.disabled),
+      isUnbounded: () => Boolean(instance.unbounded),
       registerDocumentInteractionHandler: (evtType, handler) =>
         document.documentElement.addEventListener(evtType, handler, util.applyPassive()),
-      deregisterDocumentInteractionHandler: (evtType, handler) =>
-        document.documentElement.removeEventListener(evtType, handler, util.applyPassive()),
+      registerInteractionHandler: (evtType, handler) =>
+        instance.root_.addEventListener(evtType, handler, util.applyPassive()),
       registerResizeHandler: (handler) => window.addEventListener('resize', handler),
-      deregisterResizeHandler: (handler) => window.removeEventListener('resize', handler),
-      updateCssVariable: (varName, value) => instance.root_.style.setProperty(varName, value),
-      computeBoundingRect: () => instance.root_.getBoundingClientRect(),
-      getWindowPageOffset: () => ({x: window.pageXOffset, y: window.pageYOffset}),
+      removeClass: (className) => instance.root_.classList.remove(className),
+      updateCssVariable: (varName, value) => (instance.root_ as HTMLElement).style.setProperty(varName, value),
     };
   }
 
-  /** @return {boolean} */
-  get unbounded() {
-    return this.unbounded_;
+  disabled = false;
+  root_: Element = this.root_;
```

Comment:
Possibly dumb question: is the assignment here actually doing anything or is it just assigning `this.root_` to itself?